integration_tests/e2e/index.cy.ts

```diff
@@ -1,4 +1,4 @@
-import HomePage from '../pages/home'
+import IndexPage from '../pages'
```

Comment:
Renaming various references here and in other files to the Home page back from 'home' to 'index' so that it's consistent with the template which means any future changes will come through more easily.